package.json

```diff
@@ -14,7 +14,7 @@
     "ioredis": "^4.27.1",
     "js-yaml": "^4.1.0",
     "node-fetch": "^2.6.1",
-    "puppeteer-cluster": "^0.22.0",
+    "puppeteer-cluster": "github:ikreymer/puppeteer-cluster#async-job-queue",
```

Comment:
I haven't seen this way of declaring a version before what does this do?

Reply:
Installs from a particular branch on github rather than from npm!